testsuites/kubernetes_versions.robot

```diff
@@ -130,8 +121,6 @@ Verify --wait flag works as expected
     # Delete bad release
     Should pass  helm delete wait-flag-bad
 
-Suite Setup
-    Kind.Cleanup all test clusters
```

Comment:
we want to have this available as the hook - it is fully a different question if some providers would implement this as no-op.

Reply:
Done